ee/billing/billing_manager.py

```diff
@@ -245,7 +245,7 @@ def update_org_details(self, organization: Organization, billing_status: Billing
 
         available_product_features = data.get("available_product_features", None)
         if available_product_features and available_product_features != organization.available_product_features:
-            organization.available_product_features = data["available_product_features"]
+            organization.available_product_features = available_product_features
```

Comment:
@raquelmsmith I wondered if it was possible to update mypy 🙈 

The new version was complaining here because `available_product_features` wasn't on `CustomerInfo`

I added it, but tagging you in case I've misunderstood something

Reply:
That seems like it should be fine! Does it work? 😄